### PR TITLE
Fix for requirements stuff and provisioning...also fix automated testing.

### PR DESCRIPTION
**Pull request by iheanyi**
Made some changes to requirements.txt and whatnot. Also fixed some provisioning nonsense, should fix our most crucial bugs.

### CODE REVIEW

**Comment by iheanyi** [comment]
AND YOUR BOY GOT AUTOMATED TESTING FIXING, IN YOUR FACE @reggie @GRardB, turn up~

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+database:
+  override:
+    - createuser -d -R dispiff
+    - createdb -O dispiff dispiff
```
Comment (reggie) [inline circle.yml]: why both dispiff and dispiff_test?

File: package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
     "build": "webpack --config webpack.config.js --progress --colors",
     "build-production": "webpack --config webpack.prod.config.js --progress --colors",
     "watch": "node server.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (reggie) [inline package.json]: don't forget to revert this in the future -_______-

File: provision/bootstrap.sh [hunk 3]
```diff
@@ -9,24 +9,30 @@ apt-get install -y curl
 curl --silent --location https://deb.nodesource.com/setup_0.12 | sudo bash -
 apt-get install -y nodejs
 apt-get install -y npm
-apt-get install -y build-essential
+apt-get install -y build-essential libssl-dev
```
Comment (reggie) [inline provision/bootstrap.sh]: I'm a scrub, why is libssl-dev there and why isn't the installation of it on it's own line like everything above it

File: circle.yml (hunk 1 above)
Comment (iheanyi) [inline circle.yml]: Why not? Future-proofing just in case for integration tests against actual components, I believe.

File: package.json (hunk 2 above)
Comment (iheanyi) [inline package.json]: Obviously will get updated after I finish getting the React components test suite up and running.

File: provision/bootstrap.sh (hunk 3 above)
Comment (iheanyi) [inline provision/bootstrap.sh]: I had to do it for Python and NPM stuff? I dunno, I was just following an install guide. And I don't know why it's not on its own line man, shit. Are we really gonna be this OCD about everything?

Comment (reggie) [inline provision/bootstrap.sh]: I mean... why not then just put everything on one line -.- its not OCD it's consistency. But it's fine for now

**Pull request merged**